README.md

```diff
@@ -42,7 +42,16 @@ For more information and distro-specific instructions, see the [Linux installati
 
 ### Windows
 
-`gh` is available via [scoop][], [Chocolatey][], and as downloadable MSI.
+`gh` is available via [winget][], [scoop][], [Chocolatey][], and as downloadable MSI.
+
+
+#### Windows Package Manager (aka winget)
```

Comment:
Let me know if you ant to change it to something like:

"WinGet"

or

"The Windows Package Manager"

Reply:
I think "winget" is appropriate here as it matches "scoop" and "chocolatey".

Reply:
This is how it is [mentioned](https://github.com/microsoft/terminal/#via-windows-package-manager-cli-aka-winget) in the Windows Terminal repo

I will fix it now anyways